pkg/services/ngalert/store/loki_range_to_instant_test.go

```diff
@@ -181,3 +194,125 @@ func createMigrateableLokiRule(t *testing.T, muts ...func(*models.AlertRule)) *m
 	}
 	return r
 }
+
+func createMultiQueryMigratableLokiRule(t *testing.T, muts ...func(*models.AlertRule)) *models.AlertRule {
```

Comment:
Should add some test cases in `TestMigrateLokiQueryToInstant` for multiple queries.

Reply:
Yep, added a test 👍